tests/end2end/features/yankpaste.feature

```diff
@@ -227,82 +227,59 @@ Feature: Yanking and pasting.
         And I run :open -t {clipboard}
         Then no crash should happen
 
-    #### :paste-primary
+    #### :insert-text
 
-    Scenario: Pasting the primary selection into an empty text field
-        When selection is supported
-        And I open data/paste_primary.html
-        And I put "Hello world" into the primary selection
+    Scenario: Inserting text into an empty text field
+        When I open data/paste_primary.html
         # Click the text field
         And I run :hint all
         And I run :follow-hint a
         And I wait for "Clicked editable element!" in the log
-        And I run :paste-primary
+        And I run :insert-text Hello world
         # Compare
         Then the text field should contain "Hello world"
 
-    Scenario: Pasting the primary selection into a text field at specific position
-        When selection is supported
-        And I open data/paste_primary.html
+    Scenario: Inserting text into a text field at specific position
+        When I open data/paste_primary.html
         And I set the text field to "one two three four"
-        And I put " Hello world" into the primary selection
         # Click the text field
         And I run :hint all
         And I run :follow-hint a
         And I wait for "Clicked editable element!" in the log
         # Move to the beginning and two words to the right
         And I press the keys "<Home>"
-        And I press the key "<Ctrl+Right>"
-        And I press the key "<Ctrl+Right>"
-        And I run :paste-primary
+        And I press the key "<Right>"
+        And I press the key "<Right>"
+        And I run :insert-text Hello world
         # Compare
-        Then the text field should contain "one two Hello world three four"
+        Then the text field should contain "onHello worlde two three four"
```

Comment:
Not that it's important, but why did you change the insert position? And the comment above still says "two words to the right" :wink: